webui/react/tsconfig.json

```diff
@@ -2,10 +2,8 @@
   "extends": "./src/shared/configs/tsconfig.json",
   "compilerOptions": {
     "baseUrl": "./src",
-    "module": "esnext"
+    "module": "esnext",
+    "noImplicitAny": false
```

Comment:
React18 is more strict about types. Many places don't specify types well, so I used `noImplicitAny` here. I couldn't add types all places cuz its too too many. We can add types later